bitnami/spring-cloud-dataflow/Chart.yaml

```
@@ -39,4 +39,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-spring-cloud-dataflow
   - https://github.com/bitnami/bitnami-docker-spring-cloud-skipper
   - https://dataflow.spring.io/
-version: 4.0.3
+version: 4.0.5
```

Since it's a feature addition instead of bug fixing, I would increase the minor version instead of the patch one
```suggestion
version: 4.1.0
```